dat/training.csv

```diff
@@ -0,0 +1,69 @@
+103079329921,114689,103079215232,2
+103347765377,114689,103347650688,1

```

Comment:
Also, I think we need to store an extra bit of information in here, which is the _result_ of the game (player1 win, player2 win or draw). 

The reason is, we want to train the net to evaluate which positions are good (likely to lead to a win) and bad (likely to lead to a loss). 

For each position the net makes a prediction for the outcome of the game. But we need to also know the actual outcome, in order to calculate the error margin between the net's prediction and the actual answer. 

Then we can train the net to adjust all the weights/biases to minimise the error, and it will hopefully get smarter.

how about just adding a 0, 1 or 2 as a fourth column to indicate the outcome?